manifests/repo.pp

```diff
@@ -31,6 +31,14 @@
     }
   }
 
+  if $facts['os']['release']['major'] == '8' {
```

Comment:
I think this should take the version into account. The question is what it should do if `$repo` is undef. Do we assume nightly or assume nothing. I'm leaning to the latter.